README.md

```diff
@@ -1,3 +1,13 @@
 # Hydra extensions
 
 This repository gathers vocabularies which extend [Hydra Core](http://www.hydra-cg.com/spec/latest/core/) to allow implementors to create richer hypermedia in their Hydra APIs.
+
+## Adding extensions
+
+Simply run `make new spec=my-extension` and commit the created `index.bs` file.
```

Comment:
Is it going to be an `official` way of adding extensions? I'd like it a little less formalized and opened for other approaches.

Reply:
What do you mean? Bottom line you just need to create `new-spec/index.bs` and the makefile just calls Bikeshed to initialise that for you (plus the targets to build and watch)

Reply:
What if I wanted just an old school HTML file with the spec? Or a markdown file?

Reply:
I think it's a good idea to keep the spec-like look and feel. Bikeshed and respec are two contenders and I think the former, used here, works quite well.

Reply:
OK - Bikeshed looks way bertter that Respec when using markdown